judgels-backends/sandalphon/sandalphon-api/src/main/java/judgels/sandalphon/api/submission/BundleSubmission.java

```diff
@@ -0,0 +1,20 @@
+package judgels.sandalphon.api.submission;
```

Comment:
Should this be in `bundle` package also?

Reply:
If it is separated, both `bundle` and `programming` packages will only contain 1 class each

Reply:
Will be fixed in the next pull request